tests/memory_tests/src/memory_tests/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# add dummy `memory_tests` target combines all memory tests
+add_custom_target(memory_tests)
+
+# Build test from every source file.
+# Test target name is source file name without extension.
+FILE(GLOB tests "*.cpp")
+
+add_subdirectory("${OpenVINO_SOURCE_DIR}/tests/lib" tests_shared_lib)
+
+foreach(test_source ${tests})
+    get_filename_component(test_name ${test_source} NAME_WE)
+    add_executable(${test_name} ${test_source})
+
+    target_link_libraries(${test_name} PRIVATE IE::inference_engine memory_tests_helper tests_shared_lib)
+
+    add_dependencies(memory_tests ${test_name})
```

Comment:
`install` is missed - https://github.com/openvinotoolkit/openvino/blob/master/tests/time_tests/src/timetests/CMakeLists.txt#L22